lib/ace/mode/html_completions.js

```diff
@@ -286,7 +286,7 @@ var HtmlCompletions = function() {
             
         // HTML entities
         var line = session.getLine(pos.row).substr(0, pos.column);
-        if (/&[A-z]*$/i.test(line))
+        if (/&[a-z]*$/i.test(line))
```

Comment:
Since we're already ignoring case here it probably makes sense just to just use `a-z`, unless we really do want to accept the characters between `Z` and `a`?